packages/xo-web/src/xo-app/vm-import/esxi/esxi-import.js

```diff
@@ -0,0 +1,262 @@
+          label: vm.nameLabel,
+          value: vm.id,
+        })),
+      networkpredicate:
+        ({ pool }) =>
+        network =>
+          network.$poolId === pool?.uuid,
+      srPredicate:
+        ({ pool }) =>
+        sr =>
+          isSrWritable(sr) && sr.$poolId === pool?.uuid,
+    },
+  }),
+  injectIntl,
+  injectState,
+  ({
+    effects: {
+      connect,
+      importVm,
+      linkState,
+      onChangeVm,
+      onChangeNetwork,
+      onChangePool,
+      onChangeSr,
+      reset,
+      srPredicate,
+      toggleCertificateCheck,
+    },
+    intl: { formatMessage },
+    state: {
+      defaultNetwork,
+      hasCertificate,
+      hostIp,
+      isConnected,
+      network = defaultNetwork,
+      networkPredicate,
+      password,
+      pool,
+      selectVmOptions,
+      sr,
+      user,
+      vm,
+      vmsById,
+      vmData,
+    },
+  }) => (
+    <div>
+      {!isConnected && (
+        <form id='esxi-connect-form'>
+          <Row>
+            <LabelCol>{_('hostIp')}</LabelCol>
+            <InputCol>
+              <Input
+                className='form-control'
+                name='hostIp'
+                onChange={linkState}
+                placeholder='192.168.2.20'
+                required
+                value={hostIp}
+              />
+            </InputCol>
+          </Row>
+          <Row>
+            <LabelCol>{_('user')}</LabelCol>
+            <InputCol>
+              <Input
+                className='form-control'
+                name='user'
+                onChange={linkState}
+                placeholder={formatMessage(messages.user)}
+                required
+                value={user}
+              />
+            </InputCol>
+          </Row>
+          <Row>
+            <LabelCol>{_('password')}</LabelCol>
+            <InputCol>
+              <Password
+                name='password'
+                onChange={linkState}
+                placeholder={formatMessage(messages.password)}
+                required
+                value={password}
+              />
+            </InputCol>
+          </Row>
+          <Row>
+            <LabelCol>{_('sslCertificate')}</LabelCol>
+            <InputCol>
+              <input
+                checked={hasCertificate}
+                name='hasCertificate'
+                onChange={toggleCertificateCheck}
+                type='checkbox'
+                value={hasCertificate}
+              />
+            </InputCol>
+          </Row>
+          <div className='form-group pull-right'>
+            <ActionButton
+              btnStyle='primary'
+              className='mr-1'
+              form='esxi-connect-form'
+              handler={connect}
+              icon='connect'
+              type='submit'
+            >
+              {_('serverConnect')}
+            </ActionButton>
+            <Button onClick={reset}>{_('formReset')}</Button>
+          </div>
+        </form>
+      )}
+      {isConnected && (
```

Comment:
You could use a ternary operator instead of 2 opposite conditions. That way, if we change the condition or the name of the variable, we wouldn't have to change it in 2 places.